source/includes/_subscriptions.md

```diff
@@ -439,3 +439,101 @@ size | `number` | Number of subscriptions per page _default is 10_
+infinite | `boolean` | If `true`, the subscription will run until the user stops it.
+current_billing_date_period_start | `string`| The date on which the customer was billed last.
+current_billing_date_period_end | `string`| The date on which the customer will be billed next. This will also be the date on which the next current_billing_date_period_start of the subscription starts.
+prorate_amount | `number` | The amount prorated. Currency set in [`Plan`](#Plans).
+prorate_date | `number`| The date of the last time an amount was prorated
+last_billing_amount | `number`| The amount that was subtracted at the last payment. `plan.total_amount`+`prorate_amount`
+total_fail_attempts | `number` | The number of failed payment attempts in the subscription. See [`Plan`](#Plans)`.max_fail_attempts
+initial_order | `object` | [`Order`](#Orders) ID associated with the subscription and used to create a [`Recurring Order Plan`](#recurring-order-plan)
+orders | `object` | List of [`Orders`](#Orders) 
+
+## Create a New Subscription
+To create a new subscription, the user needs to have a plan associated with the
+subscription. Additionally, the user needs to specify a subscription method.
+This will not start the subscription, only create it.
+
+> Definition
+
+```
+POST https://api.shareactor.io/subscriptions
+```
+
+
+> Example request:
+
+``` http
+POST /subscriptions HTTP/1.1
+Content-Type: application/json
+Authorization: Bearer <jwt>
+X-Share-Api-Key: <shareactor-api-key>
+Host: api.shareactor.io
+
+{
+    "plan": "5931697ed57ba271c0c7de66",
+    "subscription_method": "recurring_order"
+}
+```
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
+    "company": {"$oid": "57ee9c71d76d431f8511142f"},
+    "active": true,
+    "status": "CREATED",
+    "name": "RECURRING_ORDER",
+    "method": "recurring_order",
+    "_id": {"$oid": "5964a0ead57ba2036750a3b4"},
+    "deleted": false,
+    "prorate_amount": 0.0,
+    "_cls": "SubscriptionMethod.LicenseSubscription",
+    "plan": {"$oid": "5931697ed57ba271c0c7de66"},
+    "payments": [],
+    "infinite": false,
+    "created": {"$date": 1499767018360},
+    "user": {"$oid": "57ee9c72d76d431f85111432"},
+    "updated": {"$date": 1499767018360}
+```
+
+Attribute | Type | Description
+--------- | ---- | -------
+**plan** | `string` | ID of the [`Plan`](#Plans) associated with the subscription.
+**subscription_method** | `string` | Name of subscription method. `license`
+note | `string` | A short description.
+start_now | `boolean`| If `true`, the subscription will start now.
+
+## Start Subscription
+To start a subscription, the user needs to first create a subscription.
+The user could create and start the subscription in one API call. Then you
+will have to add `{"start_now": true}` when creating a new subscription.
+When starting the subscription, we need an associated payment_method in the request data.
```

Comment:
Take away "To start a subscription, the user needs to first create a subscription.
"

Reply:
The user could create and start the subscription in one API call. Add `{"start_now": true}` to the JSON data when creating a new subscription. A payment method needs to be set before starting a subscription.